src/IndexUpdater.php

```diff
@@ -62,7 +62,7 @@ public function updateProperties(array $properties): void
                 continue;
             }
 
-            $type = $propertyData['type'];
+            $typeOrId = $propertyData['type'];
```

Comment:
It seems confuse to me that we are using `$propertyData['type']` but calling the variable as `$typeOrId`

Maybe we should have different methods, one by type and other by ID according to the case?

Reply:
I prefer to rename back the variable for this case then because creating another method, they will be almost the same.